docs/migration/manual-migration.mdx

```diff
@@ -59,6 +58,37 @@ To cope with this change, you need to review all components using
   );
   ```
 
+## `withEntityNotFound` API change
+
+[`withEntityNotFound` API has changed](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2354).
+In version 2, the parameter `NotFoundComponent` was optional (defaulted to
+`NotFound` component). However, this HOC is now deprecated and has been placed
+within `@front-commerce/compat` package, from which it is not possible to import
+files from the `theme`. Hence in v3, `withEntityNotFound`'s `NotFoundComponent`
+parameter is now mandatory.
+
+To cope with this change, you need to review all component using
+`withEntityNotFound` to make sure a `NotFoundComponent` parameter is passed to
+it. For example, by using the component it was previously defaulted to:
+
+```jsx
+import NotFound "theme/modules/PageError/NotFound/NotFound"
+
+/*
+  ... your component
+*/
+
+export default   withEntityNotFound({
+    isFound: () => false,
+    NotFoundComponent: NotFound,
+  })(MyComponent);
+
+```
+
+A even better way would be to get rid of `withEntityNotFound`, and instead
+handle this mechanism directly withing the related `loader` by throwing a `404`
```

Comment:
```suggestion
handle this mechanism directly within the related Remix route `loader` by throwing a `404`
```
Maybe provide an example of how to throw 404 ?

Reply:
Added a link to Remix's guide on 404 ([this one](https://remix.run/docs/en/main/guides/not-found)).